run_checks.sh

```diff
@@ -1,15 +1,22 @@
-#!/bin/bash 
-if [[ ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone.jar ]]; then
-  echo "Could not find genomicsdb-${GENOMICSDB_VERSION}-allinone.jar"
-  exit 1
+#!/bin/bash
+file_size_kb=0
+if [[ ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar ]]; then
```

Comment:
Can you change the block from Line 3 onwards to something like this?
```bash
if [[ ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar && ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone.jar ]]; then
  echo "Could not find genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar or genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar"
 	exit 1
fi

if [[ -f genomicsdb-${GENOMICSDB_VERSION}-allinone.jar ]]; then
	file_size_kb=`du -k genomicsdb-${GENOMICSDB_VERSION}-allinone.jar | cut -f1`
else
	file_size_kb=`du -k genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar | cut -f1`
fi
```

Reply:
Sure, that looks good.